PrestaShop/ps_facetedsearch
# Fix missing products if default order is by manufacturer

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | If you set default order in category by manufacturer and the product has no manufacturer, its not visible at all. It is because the manufacturer table has an INNER JOIN. If you LEFT JOIN it, everything works correctly. Doesn't affect filtering.
| Type?         | bug fix
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes PrestaShop/Prestashop#30562
| How to test?  | See below or see the issue.

**How to test**
- Prepare a category, where there will be some product from manufacturer A, some products from manufacturer B and some without it.
- Go to Shop Parameters > Product Settings and set default sorting by Brand.
- Go check that category and see that the products without manufacturer disappeared.
- With my PR, everything works correctly.
- Only thing that this change could break is manufacturer filtering, checked, works correctly.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/ps_facetedsearch/754)
